packages/tempus/src/Tempus_SolutionState_impl.hpp

```diff
@@ -148,14 +148,14 @@ SolutionState<Scalar>::SolutionState(
   const Status solutionStatus,
   const bool   output,
   const bool   outputScreen,
-  const bool   isInterpolated,
   const bool   isSynced,
+  const bool   isInterpolated,
```

Comment:
This was a fortunate catch:  `isInterpolated` and `isSynced` appeared in different orders in the `_decl` and `_impl` files.  You'll want to go through and check the usage of this function everywhere to make sure the correct `bool`s are passed in in the correct order.